pkg/download.go

```diff
@@ -141,26 +155,33 @@ func NewCmdDownloadRepository(clientGetter genericclioptions.RESTClientGetter) *
 	return cmd
 }
 
-// FixIt! directly call restic/restic to restore hosts in (parallel ?)
-func runRestoreViaDocker(localDirs cliLocalDirectories) error {
+func runRestoreViaDocker(localDirs cliLocalDirectories, extraArgs []string, snapshots []string) error {
 	// get current user
 	currentUser, err := user.Current()
 	if err != nil {
 		return err
 	}
-	args := []string{
+	restoreArgs := []string{
 		"run",
 		"--rm",
 		"-u", currentUser.Uid,
-		"-v", localDirs.configDir + ":" + docker.ConfigDir,
-		"-v", localDirs.secretDir + ":" + docker.SecretDir,
+		"-v", localDirs.secretDir + ":" + localDirs.secretDir,
 		"-v", localDirs.downloadDir + ":" + docker.DestinationDir,
+		"--env-file", filepath.Join(localDirs.configDir, ResticEnvs),
 		imgRestic.ToContainerImage(),
-		"docker",
-		"download-snapshots",
 	}
-	klog.Infoln("Running docker with args:", args)
-	out, err := exec.Command("docker", args...).CombinedOutput()
-	klog.Infoln("Output:", string(out))
-	return err
+
+	restoreArgs = append(restoreArgs, extraArgs...)
+	restoreArgs = append(restoreArgs, "restore")
+
+	for _, snapshot := range snapshots {
+		args := append(restoreArgs, snapshot, "--target", filepath.Join(docker.DestinationDir, snapshot))
+		klog.Infoln("Running docker with args:", args)
```

Comment:
Make this log more clear. Specify which Snapshot from which Repository restored in which directory.